worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharSetBlocks.java

```diff
@@ -412,4 +391,33 @@ private void updateSectionIndexRange(int layer) {
         }
     }
 
+    private void extracted(int layer, int diff, char[][] tmpBlocks, Section[] tmpSections, Object[] tmpSectionLocks) {
```

Comment:
rename this to something like `resizeDataArrays`

Reply:
renamed in latest commit